cloudhub/modules/ROOT/pages/vpc-upgrade.adoc

```diff
@@ -0,0 +1,104 @@
+= CloudHub VPC to CloudHub 2.0 Private Space Upgrade  
+
+//Overview
+To migrate your workloads from CloudHub to CloudHub 2.0, you need to set up your CloudHub 2.0 infrastructure and network connections. One approach is to create new CloudHub 2.0 private spaces with configurations that differ from your existing CloudHub setup. 
+
+However, if you prefer to keep your infrastructure configuration the same, you can use the in-product VPC upgrade tool to easily migrate your CloudHub dedicated Virtual Private Cloud (VPC) to a CloudHub 2.0 private space.
+
+Once your new private space is ready, you can gradually migrate your applications from the CloudHub VPC to the CloudHub 2.0 private space. After all your applications are successfully moved, you can decommission your old VPC.
+
+
+// Benefits
+Using this VPC in-product upgrade tool saves you time by eliminating the need to acquire new CIDR blocks, set up new network connections, and configure the private space from the ground up.
```

Comment:
```suggestion
Using the in-product VPC upgrade tool saves you time because you don't need to acquire new CIDR blocks, set up new network connections, and configure the private space from the ground up.
```

Reply:
Nice benefits statement!